api/src/modules/h3-data/h3-data.service.ts

```diff
@@ -165,7 +165,7 @@ export class H3DataService {
      */
     const indicatorH3Data: H3Data | undefined =
       await this.h3DataRepository.findOne({
-        where: { indicatorId: indicatorId },
+        where: { indicatorId: indicatorId, year: year },
```

Comment:
the message on line 184 should be updated

Reply:
Thanks! Added year mentioning to the error message. This message will probably change again once we have fallback strategy implemented? Since if the material won't be found just for the year, some alternative will bee provided..